packages/provider/src/relayer/local-relayer.ts

```diff
@@ -9,6 +9,8 @@ import { addressOf } from '../utils'
 
 import { IRelayer } from '.'
 
+const DEFAULT_FEE = ethers.utils.bigNumberify(800000)
```

Comment:
can we rename this to, "DEFAULT_GAS_LIMIT"? as this isn't really the fee, its just the limit? fee would be price*limit

Reply:
Yep, good catch